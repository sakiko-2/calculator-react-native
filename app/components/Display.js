import React from 'react';
import { Text, View, StyleSheet } from 'react-native';

const Display = ({ text }) => (
  <View style={styles.container}>
    <Text style={styles.text}>{text}</Text>
  </View>
);

const styles = StyleSheet.create({
  container: {
    flex: 3,
    width: '90%',
    backgroundColor: '#000',
    justifyContent: 'flex-end',
    padding: 10
  },
  text: {
    color: '#fff',
    fontSize: 80,
    textAlign: 'right'
  }
});

export default Display;
