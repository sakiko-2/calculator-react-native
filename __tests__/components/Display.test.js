import 'react-native';
import React from 'react';
import Display from '../../app/components/Display';

import renderer from 'react-test-renderer';

describe('Display', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Display />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
