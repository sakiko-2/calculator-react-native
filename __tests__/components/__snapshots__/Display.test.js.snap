// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#000",
      "flex": 3,
      "justifyContent": "flex-end",
      "padding": 10,
      "width": "90%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 80,
        "textAlign": "right",
      }
    }
  />
</View>
`;
